src/viewer/NoViewer.jsx

```diff
@@ -1,16 +1,22 @@
 import React from 'react'
 import { connect } from 'react-redux'
-import classNames from 'classnames'
 import { translate } from 'cozy-ui/react/I18n'
-import { Button } from 'cozy-ui/react/Button'
-import Alerter from 'cozy-ui/react/Alerter'
+import { Icon, Button, Alerter } from 'cozy-ui/transpiled/react'
```

Comment:
Hmm now we are importing from the transpiled and the non transpiled one. Can we just add an alias in our webpack config ? ;)

Reply:
Talked through elsewhere: there shouldn't be a problem migrating to the transpiled version progressively, no need for an alias just yet.